modules/nw-metallb-example-addresspool.adoc

```diff
@@ -22,12 +22,12 @@ spec:
   - 192.168.100.0/24
   - 192.168.200.0/24
   - 192.168.255.1-192.168.255.5
+# ...
 ----
 
 == Example: Reserve IP addresses
```

Comment:
Modelling the "Reserve IP address" section for a reserved single IP address and multiple reserved IP addresses.